File: python/ejer25.py
opcion = 3

if opcion == 1:
    print("¡Hola! Bienvenido")

elif opcion == 2:
    print("Fecha simulada: 15/05/2026")

elif opcion == 3:

    tipoCliente = 1
    monto = 400000

    if tipoCliente == 1:

        descuento = monto * 0.20
        total = monto - descuento

        print("Cliente VIP")
        print("Descuento:", descuento)
        print("Total a pagar:", total)

    elif tipoCliente == 2:

        descuento = monto * 0.05
        total = monto - descuento

        print("Cliente Normal")
        print("Descuento:", descuento)
        print("Total a pagar:", total)

    else:
        print("Tipo de cliente inválido")

elif opcion == 4:
    print("Saliendo...")

else:
    print("Opción inválida")

File: js/ejer25.js
opcion = 3

if(opcion == 1){
    console.log("¡Hola! Bienvenido")
}

else if(opcion == 2){
    console.log("Fecha simulada: 15/05/2026")
}

else if(opcion == 3){

    tipoCliente = 1
    monto = 200000

    if(tipoCliente == 1){

        descuento = monto * 0.20
        total = monto - descuento

        console.log("Cliente VIP")
        console.log("Descuento: " + descuento)
        console.log("Total a pagar: " + total)
    }

    else if(tipoCliente == 2){

        descuento = monto * 0.05
        total = monto - descuento

        console.log("Cliente Normal")
        console.log("Descuento: " + descuento)
        console.log("Total a pagar: " + total)
    }

    else{
        console.log("Tipo de cliente inválido")
    }
}

else if(opcion == 4){
    console.log("Saliendo...")
}

else{
    console.log("Opción inválida")
}